poll/poll.py

```diff
@@ -140,53 +142,71 @@ def gather_items(self, data, result, noun, field, image=True):
             source_items = []
             result['success'] = False
             result['errors'].append(
-                "'{0}' is not present, or not a JSON array.".format(field))
+                self.ugettext("'{0}' is not present, or not a JSON array.").format(field))
```

Comment:
@asadiqbal08 This string should use a named placeholder (`field`) and is missing a translator comment, something like `# Translators: {field} is either "answers" or "questions".`